release notes/v0.37.0.md

```diff
@@ -67,7 +67,7 @@ export let options = {
             "rate=200",
             // thresholds do not support javascript expressions anymore
             "throw new Error('wat')",
-            // test is not a valid aggregation metric
+            // it fails when foo doesn't aggregate
```

Comment:
I think it was more accurate before, but instead of metric, use method instead 😸 
`// it fails, as test is not a valid threshold expression's aggregation method keyword`

Reply:
```suggestion
            // it fails, as foo is not a valid threshold expression's aggregation method keyword
```